Documentation/Configuration/Typo3ConfVars/Index.rst

```diff
@@ -148,54 +148,55 @@ may themselves be arrays.
 The configuration categories are:
 
 BE
-   :ref:`Options related to the TYPO3 CMS backend <typo3ConfVars_be>`.
+    :ref:`Options related to the TYPO3 backend <typo3ConfVars_be>`.
 
 DB
-   :ref:`Database connection configuration <typo3ConfVars_db>`.
+    :ref:`Database connection configuration <typo3ConfVars_db>`.
 
 EXT
-   :ref:`Extension installation options <typo3ConfVars_ext>`.
+    :ref:`Extension installation options <typo3ConfVars_ext>`.
 
 EXTCONF
-   Backend related language pack configuration resides here.
+    Backend-related language pack configuration resides here.
 
 EXTENSIONS
-   :ref:`Extension configuration <extension-configuration>`.
+    :ref:`Extension configuration <extension-configuration>`.
 
 FE
-   :ref:`Frontend-related options <typo3ConfVars_fe>`.
+    :ref:`Frontend-related options <typo3ConfVars_fe>`.
 
 GFX
-   :ref:`Options related to image manipulation. <typo3ConfVars_gfx>`
+    :ref:`Options related to image manipulation. <typo3ConfVars_gfx>`.
 
 HTTP
-   :ref:`Settings for tuning HTTP requests <typo3ConfVars_http>` made by TYPO3.
+    :ref:`Settings for tuning HTTP requests <typo3ConfVars_http>` made by TYPO3.
 
 LOG
-   :ref:`Configuration of the logging system <logging-configuration>`
+    :ref:`Configuration of the logging system <logging-configuration>`.
 
 MAIL
-   :ref:`Options related to the sending of emails <typo3ConfVars_gfx>`
-   (transport, server, etc.).
+    :ref:`Options related to the sending of emails <typo3ConfVars_gfx>`
+    (transport, server, etc.).
 
 SVCONF
-   :ref:`Service API configuration<services-developer-service-api-getters>`
+    :ref:`Service API configuration <services-developer-service-api-getters>`.
 
 SYS
-  :ref:`General options <typo3ConfVars_sys>` which may affect both the frontend
+    :ref:`General options <typo3ConfVars_sys>` which may affect both the
+    frontend and the backend.
 
 T3_SERVICES
-   :ref:`Service registration configuration <services-configuration-registration-changes>`
-   and the backend.
+    :ref:`Service registration configuration <services-configuration-registration-changes>`
+    and the backend.
 
 Further details on the various configuration options can be found in the
-:guilabel:`Admin Tools` as well as the TYPO3 source at
-:file:`typo3/sysext/core/Configuration/DefaultConfigurationDescription.yaml`.
-The documentation shown in the :guilabel:`Admin Tools` is automatically
+:guilabel:`Admin Tools` module as well as the TYPO3 source at
+:file:`EXT:core/Configuration/DefaultConfigurationDescription.yaml`.
```

Comment:
shall we link this on git?

Reply:
Nope, as when we link it there we have to link to a dedicated branch (or commit) - which will be wrong in some doc branches.

Reply:
I think linking such files invites readers to have a look at the code. Expecially now that the code is harder to find for newbies with the different positions. We could define a link like :github: that links to different versions according to the settings.cfg

Reply:
Okay, when this works, go for it. But this is not part of this PR.